### PR TITLE
pydeck: Support Tile3DLayer

### PR DESCRIPTION
**Pull request by ajduberstein**
For #4103 

#### Change List
-

### CODE REVIEW

**Comment by ajduberstein** [comment]
image

**Comment by ajduberstein** [comment]
Delaying this until we can make the loaders webpack externals. Otherwise the JS bundle in this change is quite large.

File: modules/jupyter-widget/src/create-deck.js [hunk 1]
```diff
@@ -8,6 +8,13 @@ import {registerLoaders} from '@loaders.gl/core';
 
 import * as deck from './deck-bundle';
 
+// See https://github.com/uber/deck.gl/blob/d74e375ddef5e2ac222641a2b93e1f4287d668f1/examples/playground/src/configuration.js#L14
+import {DracoWorkerLoader} from '@loaders.gl/draco';
+import {Tiles3DLoader, CesiumIonLoader} from '@loaders.gl/3d-tiles';
+
+// Note: deck already registers JSONLoader...
+registerLoaders([CSVLoader, DracoWorkerLoader]);
```
Comment (Pessimistress) [inline modules/jupyter-widget/src/create-deck.js:16]: @ibgreen I thought we no longer have to register the DracoLoader for 3d-tiles to work?

Comment (ibgreen) [reply modules/jupyter-widget/src/create-deck.js:16]: Yes that was the plan but sadly I didn't get that done in time for 2.1.

**Comment by ajduberstein** [comment]
> Delaying this until we can make the loaders webpack externals. Otherwise the JS bundle in this change is quite large.

The file at `modules/jupyter-widget/dist/bundle.js` appears to be 8.2 MB on both master and this branch. If no one minds, I'd like to merge this for now and figure out including loaders as webpack externals later.

**Pull request closed without merging**